runner/runners/bazel.go

```diff
@@ -135,6 +136,22 @@ func fetchBazelCommandData(bzFiler *bzsnapshot.BzFiler, cmd *runner.Command, rts
 	}
 	cmd.Timeout = d
 
+	// Adjust environment as specified by supported platform properties
```

Comment:
Does this work here? This is pre checkout in invoker - does the symlink end up in the checkout working directory for the task run? Was assuming this logic would likely live in a function here and be called immediately after the checkout channel recv<- in invoker.

Reply:
Nope you're right, totally missed that. Good catch.